terraform/helm/indexer/templates/pgproxy.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.pgproxy.enabled }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "aptos-indexer.fullname" . }}-pgproxy
+  labels:
+    {{- include "aptos-indexer.labels" . | nindent 4 }}
+data:
+  nginx.conf: |-
+{{ (tpl (.Files.Get "files/pgproxy_nginx.conf") .) | indent 4 }}
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "aptos-indexer.fullname" . }}-pgproxy
```

Comment:
looks like this is only nginx, why not just call it nginx so that it's more straight forward to understand?

Reply:
sure makes sense :)